### PR TITLE
Added node18 image

### PR DESCRIPTION
**Pull request by simaz33**
Added node18 and nodeserver18 images, Also updated relations graph.

### CODE REVIEW

File: node18/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+### Configuration
+Available binary paths for export:
+
+- /usr/bin/node
+- /usr/bin/npm
+- /usr/bin/yarn
+- /usr/bin/gulp
+- /usr/bin/grunt
+- /usr/bin/node-sass
```
Comment (TomasLesc) [inline node18/README.md:25]: Is it available, or changed to /usr/bin/sass ?

File: node18/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+version: '2.4'
+services:
+  node:
+    image: nfqlt/node16
+    volumes:
+      - './src:/home/project/src'
+      - /tmp
+
+  dev:
+    image: nfqlt/php73-dev
```
Comment (TomasLesc) [inline node18/README.md:44]: Older images are not updated, but for non-php development, we have nfqlt/dev image, example should reflect that.

**Review by TomasLesc — CHANGES_REQUESTED** [review]

File: node18/README.md (hunk 1 above)
Comment (simaz33) [reply node18/README.md]: Updated

File: node18/README.md (hunk 2 above)
Comment (simaz33) [reply node18/README.md]: Updated

**Review by TomasLesc — APPROVED** [review]

**Pull request merged**